PassleSync.Core/ContentFinders/PassleAuthorContentFinder.cs

```diff
@@ -9,8 +9,9 @@ public PassleAuthorContentFinder(ConfigService configService) : base(configServi
         {
         }
 
-        protected override string UrlPrefix => _configService.AuthorPermalinkPrefix;
-        protected override string ShortcodeName => "Shortcode";
+        protected override string UrlTemplate => _configService.PersonPermalinkTemplate;
+        protected override string ShortcodeName => "PersonShortcode";
+        protected override string SlugName => "PersonSlug";
```

Comment:
I don't think this is necessary as per previous comments as we don't need to capture this